src/CalcViewModel/GraphingCalculator/GraphingSettingsViewModel.cpp

```diff
@@ -71,6 +71,26 @@ void GraphingSettingsViewModel::InitRanges()
     m_dontUpdateDisplayRange = false;
 }
 
+void GraphingSettingsViewModel::ResetView()
+{
+    if (m_Graph != nullptr)
+    {
+        m_Graph->ResetGrid();
+        InitRanges();
+        m_XMinError = false;
+        m_XMaxError = false;
+        m_YMinError = false;
+        m_YMaxError = false;
+
+        RaisePropertyChanged("XError");
```

Comment:
For some reason this controller is written in a weird way such that it handles it's own RaisePropertyChanged, we will need to rethink this and the GraphControl implementation, not sure why the GraphControl doesn't just call SetDisplayRanges when the X/Y properties are changed there.